### PR TITLE
Buckets/files list ui enhancements

### PR DESCRIPTION
**Pull request by bryanchriswhite**

### CODE REVIEW

File: src/views/Buckets/Bucket-Files/Bucket-Details.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        </b-card-header>
+        <b-card-body>
+          <div class="row">
+            <dl class="col-md-6 mb-0">
+              <span class="row">
+                <dt class="col-sm-6">
+                  Storage:
+                </dt>
+                <dd class="col-sm-6">
+                  {{ bucket.storage }} GBMo.
```
Comment (braydonf) [inline src/views/Buckets/Bucket-Files/Bucket-Details.vue]: This value doesn't have useful value

File: src/views/Buckets/Bucket-Files/Bucket-Details.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                </dd>
+              </span>
+
+              <span class="row">
+                <dt class="col-sm-6">
+                  Public Permissions:
```
Comment (braydonf) [inline src/views/Buckets/Bucket-Files/Bucket-Details.vue]: This isn't implemented and cruft on the model

Comment (braydonf) [reply src/views/Buckets/Bucket-Files/Bucket-Details.vue]: Proposal on how such implementation can exist https://github.com/Storj/sips/pull/41/files

File: src/views/Buckets/Bucket-Files/Bucket-Details.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+              </span>
+            </dl>
+
+            <dl class="col-md-6 mb-0">
+              <span class="row">
+                <dt class="col-sm-6">
+                  Transfer:
+                </dt>
+                <dd class="col-sm-6">
+                  {{ bucket.transfer }} GB
```
Comment (braydonf) [inline src/views/Buckets/Bucket-Files/Bucket-Details.vue]: Also, these are not available and cruft on the model

File: src/views/Buckets/Bucket-Files/Bucket-Details.vue [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                </dd>
+              </span>
+
+              <span class="row">
+                <dt class="col-sm-6">
+                  Pubkeys:
```
Comment (braydonf) [inline src/views/Buckets/Bucket-Files/Bucket-Details.vue]: Same as above, cruft on the model and isn't implemented

**Comment by bryanchriswhite** [comment]
screenshot from 2017-10-24 20-13-16
screenshot from 2017-10-24 19-49-31

**Review by dylanlott — APPROVED** [review]
Looks good to me. let's test on staging and push to prod if it's good to go.

**Pull request merged**